### ISSUE TITLE
Bot crash on startup

### ISSUE THREAD

**Issue by Banonevar** [sogehige/sogeBot#3271]
### Expected behavior
Shouldn't crash
### Actual behavior
```
2020-02-17T17:51:20.511 | Bot is starting up
2020-02-17T17:51:24.586 | CURRENCY: fetching rates
2020-02-17T17:51:24.688 | CURRENCY: fetched rates
2020-02-17T17:51:26.304 !!! ERROR !!! TypeError: Cannot read property 'name' of undefined
at Users._sockets (/app/src/bot/_interface.ts:204:41)
at Timeout._onTimeout (/app/src/bot/_interface.ts:204:82)
at listOnTimeout (internal/timers.js:549:17)
at processTimers (internal/timers.js:492:7)
```
my bot is dockerized, so, i can't cat crash log.
### Additional informations

Master https://github.com/sogehige/sogeBot/commit/43f420d11f7f3937de17253fe9fc0ec9dc03c605, postgres

**Comment by sogehige** [comment]
I cannot reproduce this issue, it seems like your bot is starting so slow, that nothing is properly initialized.

**Comment by sogehige** [comment]
Should be fixed by 52dd6a3
